### DISCUSSION TITLE
How do I set up Sunbird on my local machine to contribute to the project?

### DISCUSSION THREAD

**Discussion by pranavirabatti** [Sunbird-Ed/Community#439] [General Discussion]
I need the environmentVariablesHelper.js configuration file. Can anyone help me with this?

**Comment by pranavirabatti** [comment]
Any update on this? 
@rajeevsathish

**Comment by rajeevsathish** [comment]
@pranavirabatti the env helper will have the upstream server ip address and some other data like API key and all that this will be pointing to the server instance of your sunbird Installation. so your devops team can help you on this but not the community

**Reply by pranavirabatti** [reply]
As an individual developer, I want to contribute to this project, but I have not set up any infrastructure yet. Could you please provide me with the server details that your team uses for testing purposes?
@rajeevsathish

**Reply by pranavirabatti** [reply]
Outsiders cannot contribute to this project?
